src/codegeneration/TemplateLiteralTransformer.js

```diff
@@ -283,6 +267,9 @@ export class TemplateLiteralTransformer extends TempVarTransformer {
           binaryExpression = binaryExpression.right;
       }
       let transformedTree = this.transformAny(tree.elements[i]);
+      if (element.type === TEMPLATE_SUBSTITUTION) {
+        transformedTree = parseExpression `String(${transformedTree})`;
```

Comment:
Good point.

Not sure what to do here. Doing a real ToString requires a typeof test. At this point I'm tempted to just not care about this and close this PR.